src/main/java/org/sonarlint/intellij/analysis/SonarLintAnalysisConfigurator.java

```diff
@@ -215,20 +220,21 @@ private static void configureSourceDirs(Properties properties, ModuleRootManager
     }
   }
 
-  private static void findSourceDirs(Collection<String> filesToAnalyze, Collection<String> testFolderPrefix, Collection<String> sources, Collection<String> tests) {
-    for (String f : filesToAnalyze) {
+  private static void findSourceDirs(Collection<VirtualFile> filesToAnalyze, Collection<String> testFolderPrefix, Collection<String> sources, Collection<String> tests) {
+    for (VirtualFile f : filesToAnalyze) {
+      String filePath = f.getPath();
       boolean isTest = false;
       for (String testPrefix : testFolderPrefix) {
-        if (f.startsWith(testPrefix)) {
+        if (filePath.startsWith(testPrefix)) {
           isTest = true;
           break;
         }
       }
       // TODO make relative to basedir
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)